applications/gaming.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Machine translation for gaming
+---
+
+# Gaming
+
+Machine translation for **gaming** is the translation for in-game and out-of-game text.
+
+For machine translation, gaming is challenging because the content is creative and context-dependent. In-game translation is challenging because of length restrictions.
+
+The goal of machine translation in gaming is to reach multilingual audiences. Machine translation is a fast solution for continuous content updates.
```

Comment:
> The goal of machine translation in gaming is to reach multilingual audiences.

This is too generic (applies to all translation, not just gaming) and too specific (within gaming there is a lot of variation), so it kinda says nothing.  LSP marketing talk ;-)  (And actually we want to reach monolingual audiences!)  The phrase "language barriers" is also preemptively banned from this project. ;-)

Since there are different possible goals, it's more suited to a list:

- Translate original in-game content
- Interaction between players

But it's not necessary, since it's below.

Reply:
> Machine translation is a fast solution for continuous content updates.

Could say it like this:
> Gaming content is not a fit for pure human translation if it is created or updated often or continuously streaming.